src/test/java/org/pesmypetcare/webservice/builders/PathTest.java

```diff
@@ -102,32 +104,36 @@ public void buildForumNamePath() {
     @Test
     public void buildForumNamesCollectionPath() {
         String path = Path.ofCollection(Collections.forumsNames, groupName);
-        assertEquals("groups_names/" + groupName + "/forums", path,
-            "Should return the path to the collectionof forum names.");
+        assertEquals("groups_names/" + groupName + "/forums", path
+            , "Should return the path to the collectionof forum names.");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [',' is preceded with whitespace.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5482446)